mkdocs.yml

```diff
@@ -70,16 +70,17 @@ nav:
     - OTP2 Migration Guide: 'OTP2-MigrationGuide.md'
 - Configuration:
     - Introduction: 'Configuration.md'
-    - Build: 'BuildConfiguration.md'
-    - "Boarding Locations": 'BoardingLocations.md'
-    - "Stop Area Relations": 'StopAreas.md'
-    - "Street Graph Pruning": 'IslandPruning.md'
-    - Router: 'RouterConfiguration.md'
-    - Accessibility: 'Accessibility.md'
-    - "Route Request": 'RouteRequest.md'
-    - "Realtime Updaters": 'UpdaterConfig.md'
-    - "Routing Modes": "RoutingModes.md"
```

Comment:
You didn't move this to either of these categories. Was it intentional?

Reply:
Yes. If you check the document you can see that it's hopelessly outdated. I would rather have the users read the API documentation, which has the up-to-date list of modes even if it's not prose.

Reply:
Should we have some archive folder of outdated documentation files which are not shown in the docs site?

Reply:
I've moved it into an archive folder in 989096a644d7c820957c6b836a635da939949949. Personally, I would vote to remove this file alltogether (I'm the original author) but lets discuss this at a dev meeting.

Reply:
Either works for me.